core/src/com/unciv/ui/MultiplayerScreen.kt

```diff
@@ -94,6 +94,14 @@ class MultiplayerScreen() : PickerScreen() {
         rightSideTable.add(editButton).pad(10f).row()
 
         addGameButton.onClick {
+            if (Gdx.app.clipboard.contents == null) {
```

Comment:
Maybe also check if it's a proper guid here?

Reply:
UUID.fromString(gameId.trim()) gets called inside addMultiplayerGame and would be redundant.

Reply:
Okay then
Let's put both tests in the same place, because their response should be the same, so either in the function or out of it, whatever you think is best :)